### ISSUE TITLE
Error : 'Can't resolve 'rxjs/operators/startWith'

### ISSUE THREAD

**Issue by harshi1991** [nishantmc/angular-material-fileupload#5]
## Bug
There is error when I include this in the project.

angular-material-fileupload : version 0.0.9

ERROR in
 /angular-material-fileupload/matFileUpload.esm.js
Module not found: Error: Can't resolve 'rxjs/operators/startWith' in 'D:\Code\..\..\..\..\node_modules\angular-material-fileupload'
 @ ./~/angular-material-fileupload/matFileUpload.esm.js 4:0-68
 @ ./ClientApp/app/shared.module.ts
 @ ./ClientApp/app/bankdata/bank-data.module.ts
 @ ./ClientApp/app/app.module.ts
 @ ./ClientApp/app/main.ts

##Versions
 ```
 "@angular/animations": "~4.4.5",
 "@angular/cdk": "2.0.0-beta.12",
 "@angular/common": "~4.4.5",
 "@angular/compiler": "~4.4.5",
 "@angular/core": "~4.4.5",
 "@angular/material": "2.0.0-beta.12"
```

## OS
Windows 10 Pro

**Comment by nishantmc** [comment]
Hi Harshi, do you have rxjs@5.5.0 package added to your project? rxjs is a dependency to this package.

**Comment by harshi1991** [comment]
Hi Nishant. Yes I figured it out eventually that it needs another version of rxjs. Our project uses an older version of rxjs...so unfortunately I wont be able to use this in our project. 

Thank you for the reply :) You can close this.

**Comment by nishantmc** [comment]
Sure, Thanks

**Comment by felipeleite** [comment]
Hi Nishant,

I'm having the same problem. Which version did you use to fix that? My current version is "rxjs": "^6.5.2"

Thanks.
